<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWidget</class>
 <widget class="QWidget" name="RoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="btnRefresh">
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnCreate">
     <property name="text">
      <string>创建我的房间</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>房间列表</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnExit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QListWidget" name="lwRoom"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
